2024-01-19 13:08:14,250 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-19 13:08:14,252 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Configure stats pid to 32871
2024-01-19 13:08:14,253 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-01-19 13:08:14,254 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Loading settings from /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/settings
2024-01-19 13:08:14,255 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-19 13:08:14,256 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-19 13:08:14,257 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-19 13:08:14,258 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-01-19 13:08:14,260 INFO    MainThread:32871 [wandb_init.py:_log_setup():526] Logging user logs to /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/run-20240119_130814-uxwy4iru/logs/debug.log
2024-01-19 13:08:14,261 INFO    MainThread:32871 [wandb_init.py:_log_setup():527] Logging internal logs to /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/run-20240119_130814-uxwy4iru/logs/debug-internal.log
2024-01-19 13:08:14,262 INFO    MainThread:32871 [wandb_init.py:init():566] calling init triggers
2024-01-19 13:08:14,263 INFO    MainThread:32871 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'lr': 0.001, 'loss': 'KLDivloss'}
2024-01-19 13:08:14,264 INFO    MainThread:32871 [wandb_init.py:init():591] re-initializing run, found existing run on stack: kj0lhuyd
2024-01-19 13:08:14,272 INFO    MainThread:32871 [wandb_run.py:_finish():1962] finishing run jatinsingh/kaggle-harmful-brain-activity-data_loader/kj0lhuyd
2024-01-19 13:08:14,277 ERROR   MainThread:32871 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-01-19 13:08:14,278 INFO    MainThread:32871 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-19 13:08:14,279 INFO    MainThread:32871 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-01-19 13:08:14,279 INFO    MainThread:32871 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-19 13:08:14,281 INFO    MainThread:32871 [wandb_run.py:_restore():2185] restore
2024-01-19 13:08:14,282 INFO    MainThread:32871 [wandb_run.py:_restore():2191] restore done
2024-01-19 13:08:24,495 INFO    MainThread:32871 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-01-19 13:08:24,497 INFO    MainThread:32871 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-01-19 13:08:24,515 INFO    MainThread:32871 [wandb_run.py:_footer_sync_info():3816] logging synced files
2024-01-19 13:08:26,537 INFO    MainThread:32871 [wandb_init.py:init():616] starting backend
2024-01-19 13:08:26,538 INFO    MainThread:32871 [wandb_init.py:init():620] setting up manager
2024-01-19 13:08:26,559 INFO    MainThread:32871 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-01-19 13:08:26,561 INFO    MainThread:32871 [wandb_init.py:init():628] backend started and connected
2024-01-19 13:08:26,590 INFO    MainThread:32871 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-19 13:08:26,591 INFO    MainThread:32871 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-19 13:08:26,592 INFO    MainThread:32871 [wandb_init.py:init():720] updated telemetry
2024-01-19 13:08:26,843 INFO    MainThread:32871 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-01-19 13:08:29,236 INFO    MainThread:32871 [wandb_run.py:_on_init():2254] communicating current version
2024-01-19 13:08:32,852 INFO    MainThread:32871 [wandb_run.py:_on_init():2263] got version response 
2024-01-19 13:08:32,853 INFO    MainThread:32871 [wandb_init.py:init():804] starting run threads in backend
2024-01-19 13:08:33,130 INFO    MainThread:32871 [wandb_run.py:_console_start():2233] atexit reg
2024-01-19 13:08:33,132 INFO    MainThread:32871 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-19 13:08:33,133 INFO    MainThread:32871 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-19 13:08:33,134 INFO    MainThread:32871 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-19 13:08:33,137 INFO    MainThread:32871 [wandb_init.py:init():847] run started, returning control to user process
2024-01-19 13:08:33,218 INFO    MainThread:32871 [wandb_watch.py:watch():51] Watching
2024-01-20 09:02:10,123 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-20 09:02:10,124 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Configure stats pid to 32871
2024-01-20 09:02:10,125 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-01-20 09:02:10,126 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Loading settings from /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/settings
2024-01-20 09:02:10,127 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-20 09:02:10,129 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-20 09:02:10,131 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-20 09:02:10,132 INFO    MainThread:32871 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-01-20 09:02:10,133 INFO    MainThread:32871 [wandb_init.py:_log_setup():526] Logging user logs to /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/run-20240120_090210-emxugn21/logs/debug.log
2024-01-20 09:02:10,135 INFO    MainThread:32871 [wandb_init.py:_log_setup():527] Logging internal logs to /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/run-20240120_090210-emxugn21/logs/debug-internal.log
2024-01-20 09:02:10,136 INFO    MainThread:32871 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fb66c7ad550>
2024-01-20 09:02:10,144 INFO    MainThread:32871 [wandb_init.py:init():566] calling init triggers
2024-01-20 09:02:10,145 INFO    MainThread:32871 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'lr': 0.001, 'loss': 'KLDivloss'}
2024-01-20 09:02:10,147 INFO    MainThread:32871 [wandb_init.py:init():591] re-initializing run, found existing run on stack: uxwy4iru
2024-01-20 09:02:10,160 INFO    MainThread:32871 [wandb_run.py:_finish():1962] finishing run jatinsingh/kaggle-harmful-brain-activity-data_loader/uxwy4iru
2024-01-20 09:02:10,162 INFO    MainThread:32871 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-20 09:02:10,166 INFO    MainThread:32871 [wandb_run.py:_restore():2185] restore
2024-01-20 09:02:10,168 INFO    MainThread:32871 [wandb_run.py:_restore():2191] restore done
2024-01-20 09:02:21,965 INFO    MainThread:32871 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-01-20 09:02:21,967 INFO    MainThread:32871 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-01-20 09:02:21,998 INFO    MainThread:32871 [wandb_run.py:_footer_sync_info():3816] logging synced files
